index.bs

```diff
@@ -7074,6 +7077,26 @@ To <dfn>get the browsing context</dfn> with given |realm|:
 
 1. Return |document|'s [=/browsing context=].
 
+</div>
+<div algorithm>
+To <dfn>get the worker's owners</dfn> with given |global object|:
+
+1. Assert: |global object| is a {{WorkerGlobalScope}} object.
+
+1. Let |owners| be an empty [=/list=].
+
+1. For each |owner| in the |global object|'s associated [=owner set=]:
+
+   1. Let |owner environment settings| be the [=environment settings object=]
```

Comment:
I know this is copied from elsewhere in the spec, but looking at HTML now it seems like one should use https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-global (since relevant global object is only really defined for platform objects, and environment settings objects aren't platform objects).

Reply:
Done.